install.sh

```diff
@@ -94,133 +103,128 @@ esac
 
 # functions to maintain packages
 function download_check () {
-    cd $CREW_BREW_DIR
+    cd "${CREW_BREW_DIR}"
 
     #download
-    echo "Downloading $1..."
-    curl --progress-bar -C - -# -L --ssl $2 -o "$3"
+    echo "Downloading ${1}..."
+    curl --progress-bar -C - -L --ssl "${2}" -o "${3}"
 
     #verify
-    echo "Verifying $1..."
-    echo $4 $3 | sha256sum -c -
-    case $? in
+    echo "Verifying ${1}..."
+    echo "${4}" "${3}" | sha256sum -c -
+    case "${?}" in
```

Comment:
This should always return a number so quoting is not necessary.  In fact, since `0` is used below, it may not work.

Reply:
Quotes aren't taken into account in the `case` statement itself. It's just that it consolidates the value of `${?}` into one argument, if it contains spaces. I'm not taking any chance here.